grakn/service/Session/util/ResponseReader.py

```diff
@@ -294,7 +300,7 @@ def _create_concept_map(tx_service, grpc_concept_map_msg):
     @staticmethod
     def _create_answer_group(tx_service, grpc_answer_group):
         grpc_owner_concept = grpc_answer_group.owner
-        owner_concept = ConceptFactory.create_concept(tx_service, grpc_owner_concept)
+        owner_concept = ConceptFactory.create_remote_concept(tx_service, grpc_owner_concept)
```

Comment:
do we require remote concepts here?

Reply:
I'm not sure but I haven't done any analysis on what might be necessary to pre-fetch here, so this is how we currently do it in Java.

Reply:
ok, can you try using a local concept and see if it breaks any tests? It shouldn't need any remote functionality...